src/Bicep.Core.UnitTests/Assertions/StringAssertionsExtensions.cs

```diff
@@ -85,6 +86,33 @@ public static AndConstraint<StringAssertions> EqualIgnoringNewlines(this StringA
             return new AndConstraint<StringAssertions>(instance);
         }
 
+        /// <summary>
+        /// Compares two strings after normalizing by unindenting lines until the least indented line is flushed left, similar to
+        /// YAML blocks of text.
+        /// </summary>
+        public static AndConstraint<StringAssertions> EqualIgnoringMinimumIndent(this StringAssertions instance, string? expected, string because = "", params object[] becauseArgs)
+        {
+            var normalizedActual = instance.Subject is null ? null : StringTestUtils.Unindent(StringUtils.ReplaceNewlines(instance.Subject, "\n"));
+            var normalizedExpected = expected is null ? null : StringTestUtils.Unindent(StringUtils.ReplaceNewlines(expected, "\n"));
+
+            normalizedActual.Should().Be(normalizedExpected, because, becauseArgs);
+
+            return new AndConstraint<StringAssertions>(instance);
+        }
+
+        /// <summary>
+        /// Compares two strings after normalizing by removing whitespace from the beginning and ending of all lines
+        /// </summary>
+        public static AndConstraint<StringAssertions> EqualIgnoringIndent(this StringAssertions instance, string? expected, string because = "", params object[] becauseArgs)
```

Comment:
[nit] Should this be called `EqualTrimmed` instead? `EqualIgnoringIndent` sounds like it would only trim whitespace from the start of each line.

Reply:
Thanks, was struggling for a better name.